src/Aspire.Hosting.Azure.CognitiveServices/AzureOpenAIDeployment.cs

```diff
@@ -9,10 +9,14 @@ namespace Aspire.Hosting.ApplicationModel;
 /// <param name="name">The name of the deployment</param>
 /// <param name="modelName">The name of the model.</param>
 /// <param name="modelVersion">The version of the model.</param>
-/// <param name="skuName">The name of the SKU.</param>
-/// <param name="skuCapacity">The capacity of the SKU.</param>
-public class AzureOpenAIDeployment(string name, string modelName, string modelVersion, string skuName = "Standard", int skuCapacity = 8)
+public class AzureOpenAIDeployment(string name, string modelName, string modelVersion)
```

Comment:
Would it be "less breaking" if we did:

```suggestion
public class AzureOpenAIDeployment(string name, string modelName, string modelVersion, string? skuName = null, int? skuCapacity = null)
```